src/Element/TraversableElement.php

```diff
@@ -157,7 +157,11 @@ public function fillField($locator, $value)
             throw new ElementNotFoundException($this->getDriver(), 'form field', 'id|name|label|value|placeholder', $locator);
         }
 
-        $field->setValue($value);
+        if ($field->getTagName() === "select") {
```

Comment:
well, they should, which is precisely why I said that the testsuite needs to cover it, to see whether the functional test passes without the change when running it in drivers